From f7f765562563e3b739f2e57b26a32316d64c5ca2 Mon Sep 17 00:00:00 2001
From: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
Date: Wed, 30 Sep 2020 17:58:43 +0530
Subject: [PATCH 1684/1851] pinctrl: zynqmp: Remove unused variable zynqmp_pins

commit 87e48f40f2ee07d7eef62abf3ef730ab29c40708 from
https://github.com/Xilinx/linux-xlnx.git

Removed unused variable "zynqmp_pins", as it is declared but never
referenced in the driver.

Addresses-coverity: Parse warning (PW.DECLARED_BUT_NOT_REFERENCED).
Signed-off-by: Sai Krishna Potthuri <lakshmi.sai.krishna.potthuri@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/pinctrl/pinctrl-zynqmp.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/pinctrl/pinctrl-zynqmp.c b/drivers/pinctrl/pinctrl-zynqmp.c
index c76cb6b5ad3f..e7e672afcd0d 100644
--- a/drivers/pinctrl/pinctrl-zynqmp.c
+++ b/drivers/pinctrl/pinctrl-zynqmp.c
@@ -108,7 +108,6 @@ pin_config_item zynqmp_conf_items[ARRAY_SIZE(zynqmp_dt_params)] = {
 #endif
 
 static const struct zynqmp_eemi_ops *eemi_ops;
-static const struct pinctrl_pin_desc zynqmp_pins;
 static struct pinctrl_desc zynqmp_desc;
 
 /**
-- 
2.31.1

